src/Tools/MicroBuild/copy-symbols.ps1

```diff
@@ -0,0 +1,35 @@
+# Copy our symbols to the Symbols directory.  This simplifies the logic of our achiving 
+# and indexing steps.  
+Param(
+    [string]$binariesPath = $null
+)
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try
+{
+    $symbolsPath = join-path $binariesPath "Symbols"
```

Comment:
In order to actually send this stuff downstream we have to have it in "symbol store" form -- the Symbols folder can't be simple flat folder like this. We _can_ process this with a microbuild task but then I think we want this name to be something else so Symbols is reserved for the existing format.

The ultimate "nice place" to be would be to have the Symbols folder be created directly as pointers. There's just a simple app that you could invoke if you feel like playing with it: https://msdn.microsoft.com/en-us/library/windows/desktop/ms681417(v=vs.85).aspx

But short term, I think let's call this FlatSymbols or something and the other name is for the other form that is unchanged from now.